From 0c461d59cf848bdbabafcc823629628ce30c09d7 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Fri, 21 Aug 2015 15:57:16 +0800
Subject: [PATCH 0271/5242] MLK-11395-6 ARM: dts: imx7d-12x12-lpddr3-arm2: add
 dts support for mipi dsi

commit  f50b20a3714cdb3230afe371f439cbec3b1fd207 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add mipi dsi dts support for imx7d-12x12-lpddr3-arm2 board.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts |   10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
index 8fb5d83..c46dbce 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
@@ -69,6 +69,13 @@
 			gpio = <&gpio1 7 GPIO_ACTIVE_HIGH>;
 			enable-active-high;
 		};
+
+		reg_mipi_dsi_pwr_on: mipi_dsi_pwr_on {
+			compatible = "regulator-fixed";
+			regulator-name = "mipi_dsi_pwr_on";
+			gpio = <&gpio4 16 GPIO_ACTIVE_HIGH>;
+			enable-active-high;
+                };
 	};
 
 	memory {
@@ -326,6 +333,9 @@
 				MX7D_PAD_SD2_WP__GPIO5_IO10       0x17059
 				MX7D_PAD_SD2_RESET_B__GPIO5_IO11  0x17059
 				MX7D_PAD_GPIO1_IO12__SD2_VSELECT  0x17059
+
+                                MX7D_PAD_ECSPI1_SCLK__GPIO4_IO16  0x59
+                                MX7D_PAD_ECSPI1_MOSI__GPIO4_IO17  0x59
 			>;
 		};
 
-- 
1.7.9.5

